### DISCUSSION TITLE
translation/istio-spire-integration/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#27] [Announcements]
# translation/istio-spire-integration/

Istio 1.14 版本增加了对 SPIRE 集成的支持，这篇文章将指导你如何在 Istio 中集成 SPIRE。

http://localhost:1313/translation/istio-spire-integration/

**Comment by rootsongjc** [comment]
在我看来，Istio 1.14 版本中最值得关注的特性是 Istio 新增对 SPIRE 的支持。
